finance/models.py

```diff
@@ -88,7 +89,11 @@ class Benefactor(models.Model):
     benefactor_locality = models.ForeignKey(
         'locality.Locality', null=True, default=None)
 
+    def __str__(self):
+        return self.benefactor_type
```

Comment:
Is there a point to defining this method as well as mixing in `ReverseLookupStringMixin` (which seemingly mixes in the same method)?